app/src/main/java/ch/sdp/vibester/activity/WelcomeActivity.kt

```diff
@@ -66,6 +73,12 @@ class WelcomeActivity : AppCompatActivity() {
     }
 
     fun switchToSearch(view: View) {
```

Comment:
I believe the user should still access the user search, but the ability to add users with friends will be disabled. I will fix it in the next pr.